scripts/new_project.kts

```diff
@@ -38,14 +39,32 @@ object NewProject {
         println("\n${message}\n")
     }
 
-    private fun copyFiles(fromPath: String, toPath: String) {
-        val targetFolder = File("../$toPath")
-        val sourceFolder = File("../$fromPath")
+    private fun copyFiles(fromPath: String) {
+        val targetFolder = File(getRootPathOfNewProjectFolder())
+        val sourceFolder = File(getRootPathOfNewProjectFolder().replace(appNameWithoutSpace, "") + fromPath)
         sourceFolder.copyRecursively(targetFolder, true) { file, exception ->
             showMessage(exception?.message ?: "Error copying files")
             return@copyRecursively OnErrorAction.TERMINATE
         }
     }
+
+    private fun cleanNewProjectFolder() {
+        executeCommand("sh " + getRootPathOfNewProjectFolder() + File.separator + "gradlew -p " + getRootPathOfNewProjectFolder() + " clean")
+        executeCommand("sh " + getRootPathOfNewProjectFolder() + File.separator + "gradlew -p " + getRootPathOfNewProjectFolder() + "/buildSrc clean")
+        File(getRootPathOfNewProjectFolder() + File.separator + ".idea")?.deleteRecursively()
+        File(getRootPathOfNewProjectFolder() + File.separator + ".gradle")?.deleteRecursively()
+        File(getRootPathOfNewProjectFolder() + File.separator + "buildSrc" + File.separator + ".gradle")?.deleteRecursively()
+        File(getRootPathOfNewProjectFolder() + File.separator + ".git")?.deleteRecursively()
```

Comment:
can we refactor these lines to avoid duplicated code please? 🙏 
```suggestion
        listOf(".idea", ".gradle", "buildSrc" + File.separator + ".gradle", ".git").forEach {
            val targetFile = File(getRootPathOfNewProjectFolder() + File.separator + it)
            if (targetFile.exists()) targetFile.deleteRecursively()
        }
```

Reply:
@manh-t Updated 🚀